threading/client/client.py

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/python3
+import socket
+import sys
+import os
+import argparse
+import signal
+import struct
+
+'''
+GLOBAL VARIABLE: FILE_LIST
```

Comment:
This is not proper module docstring format prescribed in the Google Python style guide: https://google.github.io/styleguide/pyguide.html#382-modules